editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -2128,7 +2128,12 @@ void SpatialEditorViewport::_notification(int p_what) {
 			}
 		}
 
-		_update_camera(delta);
+		if (EditorNode::get_singleton()->is_update_camera_without_interpolation()) {
```

Comment:
It's overkill to add a new property just for that. I think `EditorNode::get_singleton()->is_changing_scene()` would work just fine.

Reply:
Unfortunately this method doesn't work because of this line: https://github.com/godotengine/godot/blob/d7b9fcd33689673c6237d91afdb4a8dfe548784a/editor/editor_node.cpp#L2641
and I was afraid to move this and break something.

Reply:
I'll take a closer look.

Reply:
@akien-mga I found another way to do this with less code. What do you think?

Reply:
Less code indeed, but re-using the notification looks like a hack (since visibility didn't strictly change). Also, it looks like this won't work if multi-viewport modes are used (split/quad views).